Gem/Code/Source/Components/NetworkWeaponsComponent.cpp

```diff
@@ -16,6 +17,9 @@
 namespace MultiplayerSample
 {
     AZ_CVAR(bool, cl_WeaponsDrawDebug, true, nullptr, AZ::ConsoleFunctorFlags::Null, "If enabled, weapons will debug draw various important events");
+    AZ_CVAR(float, cl_WeaponsDrawDebugSize, 0.25f, nullptr, AZ::ConsoleFunctorFlags::Null, "The size of sphere to debug draw during weapon events");
+    AZ_CVAR(float, cl_WeaponsDrawDebugDurationSec, 10.0f, nullptr, AZ::ConsoleFunctorFlags::Null, "The number of seconds to display debug draw data");
+    AZ_CVAR(float, sv_ImpulseScalar, 750.0f, nullptr, AZ::ConsoleFunctorFlags::Null, "A fudge factor for imparting impulses on rigid bodies due to weapon hits");
```

Comment:
sv_ImpulseScalar = sv_WeaponsImpulseScalar (just for consistency?)  I sort of wish all our cl_ and sv_ was prefixed with a net_ or something, this way i could search net_ for all the useful network cvars or net_sv/net_cl to narrow it down

Reply:
Good call, done.